mff_rams_plugin/templates/kiosk_printing/print_badges.html

```diff
@@ -2,7 +2,7 @@
 <head>
     <!-- Print and Reload Function -->
     <script type="text/javascript">
-        window.print();
+        //window.print();
         // reload to print the next badge
         setTimeout(function() {window.location = "print_badges?minor={{ minor }}&evenOdd={{ evenOdd }}";}, 9000);
```

Comment:
update this to 3500ms while you're at it.

Reply:
and done.